geode-core/src/main/java/org/apache/geode/cache/client/internal/PoolImpl.java

```diff
@@ -921,7 +921,7 @@ public void returnConnection(Connection conn) {
    * Test hook that acquires and returns a connection from the pool with a given ServerLocation.
    */
   public Connection acquireConnection(ServerLocation loc) {
-    return manager.borrowConnection(loc, false);
+    return manager.borrowConnection(loc, 15000L, false);
```

Comment:
The other acquireConnection() method uses a hard-coded timeout of 45000L, though I don't know why that is.  Why does this version use 15000L? We shoudl remove the "test hook" comment because this method is used in GatewaySenderEventRemoteDispatcher.initializeConnection().

Reply:
Thanx for comments.
I find that previous code contained this value 15000L (which is removed with GEODE-6535).